Engine/source/console/simEvents.cpp

```diff
@@ -129,7 +129,7 @@ ConsoleValueRef SimConsoleThreadExecCallback::waitForResult()
       return retVal;
    }
 
-   return (const char*)NULL;
+   return ConsoleValueRef();
```

Comment:
I'm guessing no but it doesn't make sense to return retVal if for whatever reason acquire fails.